app/src/main/res/layout/login.xml

```diff
@@ -94,10 +94,18 @@
     </com.google.android.gms.common.SignInButton>
 
     <TextView
+        android:id="@+id/local_server_button"
```

Comment:
is there a way to toggle this for non-dev builds?  right now it seems that this will appear on production builds (which is a no go).